website/docs/quickstarts/duckdb-core-qs.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Quickstart for dbt Core using DuckDB"
+id: duckdb
+description: "Install dbt Core and DuckDB to begin testing within minutes"
+platform: 'dbt-core'
+icon: 'fa-light fa-square-terminal'
+hide_table_of_contents: true
+---
+
+## Introduction
+
+In this quickstart guide, you’ll learn how to install dbt Core and DuckDB, and import sample data from the fictional e-commerce Jaffle Shop data set within minutes. DuckDB is an in-process SQL OLAP database management system optimized for data analytics. Installing dbt Core and DuckDB together allows you to test all of the available features and functionality dbt has to offer locally on your computer (no data warehouse required), and the Jaffle Shop provides a pre-built dbt project ready to run out of the box. 
+
+## Prerequisites
+
+* To use the dbt CLI, you must know some command line basics. In particular, you should understand `cd`, `ls` and `pwd` to navigate through the directory structure of your computer.
+* A Windows, Mac, or Linux computer with sufficient storage. The Jaffle Shop sample data set is small and lightweight, taking up less than 200MB. 
```

Comment:
Will leave it in since the other guides are written this way but may be worth considering removing "you must have" to make it less wordy, since that's assumed since these are prerequisites